Day-I-Social-Card/social-card/src/components/HeaderComponents/ImageThumbnail.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Header.css';
+import '../../App.css';
```

Comment:
You don't have to include the same `css` file in every component. If it is included . once in the root component, it is present everywhere.